recipes/musdb/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "musdb" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/sigsep/sigsep-mus-db.git
+    git_tag: "v{{ version }}"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - libsndfile
+    - ffmpeg
+    - setuptools
+    - numpy >=1.7
+    - pysoundfile >=0.9.0
+    - stempeg >=0.1.7
```

Comment:
This package ``stempeg`` does not exist in ``conda-forge``. You will need to package it first

Reply:
Yes, I have seen this with CI failure. I am currently packaging it and will submit recipe quite soon. Will let you know as long as it is done ;).

Reply:
Just created PR for `stempeg` package : https://github.com/conda-forge/staged-recipes/pull/9998 and need review :)